cpp/tensorrt_llm/nanobind/batch_manager/cacheTransceiver.cpp

```diff
@@ -179,9 +180,11 @@ void tb::CacheTransceiverBindings::initBindings(nb::module_& m)
             nb::arg("input"), nb::arg("sizes"));
 
     nb::class_<tb::kv_cache_manager::CacheTransBufferManager>(m, "CacheTransBufferManager")
-        .def(nb::init<tb::kv_cache_manager::BaseKVCacheManager*, std::optional<size_t>>(), nb::arg("cache_manager"),
-            nb::arg("max_num_tokens") = std::nullopt)
+        .def(nb::init<tb::kv_cache_manager::BaseKVCacheManager*, std::optional<size_t>,
```

Comment:
It's a bit confusing here why one API uses tle::DataType and another one uses the TRT type.